src/screens/User.tsx

```diff
@@ -2,18 +2,20 @@ import React, { useState } from "react";
 import { Text, View, TouchableOpacity, Image } from "react-native";
 import Layout from "../components/Layout";
 import ProfileCategory from "../components/ProfileCategory";
-import Profile from "../components/Profile";
+import ProfileForm from "../components/ProfileForm";
 import Orders from "../components/Orders";
+import Discounts from "../components/Discounts";
 
 function getPageContent(category: number) {
   switch (category) {
     case 0:
```

Comment:
category should be a number? kinda unreliable to have it like this. maybe its better to have an enum only for categories

```ts
enum Categories = {
  "Profile": 0,
  "Orders": 1,
  "Discounts": 2,
}
```

and then you can use it

```ts
  case Categories.Profile: 
```

not sure this works but it can be worked around

Reply:
Ok. I have to start using `enums` more often